may-2018-hardfork.md

```diff
@@ -0,0 +1,33 @@
+# May 2018 Hardfork Specification
+
+Version Draft 0.2, 2018-02-01
+
+## Summary
+ 
+When the median time past[1] of the most recent 11 blocks (MTP-11) is greater than or equal to UNIX timestamp 1526400000 Bitcoin Cash will execute a hardfork according to this specification. Starting from the next block these consensus rules changes will take effect:
+
+* Blocksize increase to 32 MB
+* Re-enabling of several opcodes
+
+The following are not consensus changes, but are recommended changes for Bitcoin Cash implementations:
+
+* Automatic replay protection for future hardforks
+* Increase OP_RETURN relay size to 223 total bytes
```

Comment:
Maybe this should be mad explicit that it means a payload of 220 bytes, 223 being the total size.

Reply:
To be pedantic: the actual payload size really depends on the number of push operations used, which also count in terms of this limit. The +3 overhead comes from the opcode and two pushes and is inherited from https://github.com/bitcoin/bitcoin/pull/6424.